app/features/build/build_controller.rb

```diff
@@ -58,10 +64,13 @@ class BuildController < AuthenticatedControllerBase
       end
     end
 
+    # Since this controller is now not protected by authorization
+    # we might want to consider protecting this by ensuring we can only call it with
+    # the current project somehow?
```

Comment:
Should this be a `# TODO` item or GitHub issue maybe?

Reply:
I'm ok with just this comment. It's a thought. No action needed right now.